tests/ci/runpytest.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
```

Comment:
should add the standard copyright info at the top

Reply:
done